recipes/cantools/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cantools" %}
+{% set version = "20.6.0" %}
+{% set sha256 = "1cc876ec1c8de17bafc6f631292d9d71902da06aae6a394ec6fc3f4f5518389f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - cantools=cantools.__init__:_main
+
+requirements:
+  build:
+    - python
+    - pip
+    - bitstruct>=3.7.0
+    - pyparsing>=2.0.3
+    - python-can>=2.1.0
```

Comment:
Are all of these build deps?

Reply:
Also it seems that we don't have python-can in conda-forge, would you be interested in adding it?